### PR TITLE
[Term entry] Python sets .issuperset()

### LINKED ISSUE #4614: [Term Entry] Python sets .issuperset()
### Reason/inspiration (optional)

We would like a new term entry on the '.issuperset()' method for Sets in Python. The entry should go in a new file under `docs/content/python/concepts/sets/terms/issuperset/issuperset.md`

The entry should include the following:

- A description of the function
- A ## Syntax section that includes the syntax of the function and its details
- An ## Example section that includes a ```python ``` codeblock and ```shell ``` that shows the example output.
- A ## Codebyte Example that includes a ```codebyte/python ``` runnable codeblock

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ULTRAPAIN**
### Description
I have Added the Term issuperset() method in `docs/content/python/concepts/sets/terms/issuperset/issuperset.md`
Added all of the content down below
- [x] A description of the function
- [x] A ## Syntax section that includes the syntax of the function and its details
- [x] An ## Example section that includes a python  codeblock and shell  that shows the example output.
- [x] A ## Codebyte Example that includes a codebyte/python  runnable codeblock

### Issue Solved
Closes #4614

### Type of Change
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
Hey, @ULTRAPAIN I'll review the PR, but please make sure that you do not submit a PR unless the issue is assigned to you. As I can see this issue is not assigned to you.

Also, this particular entry has two files on the same branch, could you remove the `.isdisjoint` entry from this branch and keep only the required one?
Thank you!

**Comment by ULTRAPAIN** [comment]
Hi @mamtawardhani I have reverted the commit

File: content/python/concepts/sets/terms/issuperset/issuperset.md
Comment (mamtawardhani) [inline content/python/concepts/sets/terms/issuperset/issuperset.md:14]:
```suggestion
---
Title: '.issuperset()'
Description: 'Checks whether all elements of the specified set exist in the original set.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Collections'
  - 'Methods'
  - 'Sets'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

File: content/python/concepts/sets/terms/issuperset/issuperset.md
Comment (mamtawardhani) [inline content/python/concepts/sets/terms/issuperset/issuperset.md:16]:
```suggestion
In Python, the **`.issuperset()`** method is used to determine if the original set contains all elements of the specified set. It returns `True` if every element of the specified set is also an element of the original set, and `False` otherwise. 
```

File: content/python/concepts/sets/terms/issuperset/issuperset.md
Comment (mamtawardhani) [inline content/python/concepts/sets/terms/issuperset/issuperset.md:28]: This can be added in a note format, but make sure that you add it below the parameter description, use the following:

>**Note:** In Python, the `>=` operator can be used to check if one set is a superset of another, functioning equivalently to the `.issuperset()` method.

File: content/python/concepts/sets/terms/issuperset/issuperset.md
Comment (mamtawardhani) [inline content/python/concepts/sets/terms/issuperset/issuperset.md:31]:
``` suggestion
- `set1`: The set that is being examined to determine if its elements are all present in `set2`.
- `set2`: The set that is being checked to see if it contains all elements from `set1`.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Thank you for Contributing to Codecademy Docs @ULTRAPAIN! The entry was well written.

I've requested a few changes in the entry, could you please review and modify those at your earliest convenience? 
Thank you 😄

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @ULTRAPAIN!

Looks good for the second review 🚀

**Review by Sriparno08 — APPROVED** [review]
Good job, @ULTRAPAIN! 👍

**Pull request merged**
